.github/workflows/cd.yml

```diff
@@ -1,5 +1,6 @@
 name: CD
 on:
+  workflow_dispatch:
```

Comment:
Not sure this'll have much benefit over the current scheme of running on `push`. I think this change will mean we can deploy what's currently deployed, on demand.

Reply:
Does the current configuration allow us to  deploy _any_ branch?

Reply:
Re-read the PR description and it sounds like we want to be able to deploy non-default branches to PROD? Any reason why, as this doesn't follow the majority of other apps. I think we'd want to tweak the [branches rule](https://github.com/guardian/editions-card-builder/blob/8ac482ece01cd92f91e6ac07b49c1038ca133245/.github/workflows/cd.yml#L3-L5) too.

Reply:
I suppose in the absence of a CODE stack it can be useful to deploy something to PROD from a development branch for temporary testing. The use case I came across was that I wanted Katy to test the new cover card crop sizes, but if it didn't work I needed to be able to revert quickly. Perhaps the ideal would have been:

 - get approval for change, merge
 - test, it doesn't work
 - do revert of PR
 - (wait for someone to approve revert)
 - merge revert

but the testing was taking place fairly early in the morning when people had standups etc, so hard to guarentee that  someone will be around. With this change you could:

- deploy dev branch to PROD
- test, it doesn't work
- deploy master back to PROD

Arguably, we should just do our testing in a time when there's support from others to review PRs!  So happy for this to be closed, I think it's a useful functionality to have though even if it's a "use with caution" thing

Reply:
I guess the real solution here would be to have a CODE environment, but that's more trouble than it's worth!

Reply:
>  I guess the real solution here would be to have a CODE environment, but that's more trouble than it's worth!

In lieu of that, this feels like it simplifies the testing workflow and makes it clearer when we are actually testing something from a particular branch, as opposed to potentially polluting the commit history with a number of merges and reverts.

Reply:
As someone who's run into the merge/revert cycle 3 times now I agree with this! I think this would be a useful change, but also happy to close this PR if it doesn't seem like a good idea.

Reply:
I think this is a useful change to make. If we merge it, it might be useful to have something in the README about how to use this approach.